### DISCUSSION TITLE
Getting "Hmm. We’re having trouble finding that site." or “This Site Can’t Be Reached”

### DISCUSSION THREAD

**Discussion by beneix** [AdguardTeam/AdGuardHome#7112] [General]
I've been running AdGuard Home on my OPNSense router for a couple of years and it's been working great. Lately, however, I have started having issues on PCs connected to the router. In both Vivaldi and Firefox, I get messages that a site cannot be reached. On a page refresh, 9 times out of 10 the page displays. If I set the client to bypass Adguard the problem goes away.

I think the problem arose with v0.107.51 and I am now running v0.107.52. There may be something else going on, but that is when I first started having the issue.

Before I report it as an issue, what can I do to resolve it? I have already uninstalled and reinstalled the plugin on my OPNSense router and also rebooted the router.

**Comment by beneix** [comment]
I also get "Update check failed. Please check your Internet connection." on the AdGuard Home dasbhoard. One or two refreshes of the page gets the same result, then it manages to connect and tells me AdGuard Home is up-to-date and subsequent refreshes consistently work. This seems to be the same issue - something goes wrong on initial connection, but second or third attempt succeeds.

No idea what else I can check? Any log file to look at?

**Reply by trailwhim** [reply]
I'm having this problem. Did you ever find a resolution?

**Reply by beneix** [reply]
Hi - no, it has not been resolved yet. I suspect it may be related to [this issue](https://github.com/AdguardTeam/AdGuardHome/issues/6818), which seems to be caused by network contention for DNS lookups; my suspicion is that lookups randomly fail due to the contention issue. It seems to be fixed in the edge release, I am waiting for the next stable release to install that.

**Comment by beneix** [comment]
@trailwhim for me, this is resolved in the current beta and edge releases. Hopefully, the next stable release will therefore fix it for you.
